### PR TITLE
FEAT/adding pickle into main.py

### PR DESCRIPTION
**Pull request by michhottinger**
## Types of Changes

I added the pickle from the model notebook into the main.py so that the model can be applied to new incoming data.
https://trello.com/c/iOSrB5Oj
Please review the location I placed the pickle in main.py.

- [x ] I wrote a label and description of the requested changes
- [x ] PR was sent to multiple reviewers, including at least one team member and the TPL
- [x ] I performed a self-review of my own code (followed the code review checklist)
- [ x] I linked to the user story/Trello checklist item corresponding to the PR

### CODE REVIEW

**Comment by maiali13** [comment]
I see that the ` from sklearn.preprocessing import MultiLabelBinarizer` statement is at the top of the file! It seems like the issue where it was getting auto deleted for some reason has been resolved.
